### PR TITLE
New Maintainer for Xiaomi MI A3 (laurel_sprout)

### PR DESCRIPTION
**Pull request by RainySorcerer**

### CODE REVIEW

File: laurel_sprout.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "RainySorcerer (RainZ)",
+			"oem": "xiaomi",
```
Comment (gwolf2u) [inline laurel_sprout.json:5]: Use capital starting letter X

File: laurel_sprout.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+			"device": "laurel_sprout",
```
Comment (gwolf2u) [inline laurel_sprout.json:6]: Device name, not codename

File: laurel_sprout.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/development-rom-14-0-crdroid-10-unofficial.4695645/",
+			"gapps": "https://sourceforge.net/projects/nikgapps/files/Elite-Releases/crdroid-official/Android-14",
+			"firmware": "",
+			"modem": "",
+			"bootloader": "",
+			"recovery": "",
+			"paypal": "",
+			"telegram": "https://t.me/MysticZyx",
+			"dt": "https://github.com/crdroidandroid/android_device_xiaomi_laurel_sprout",
```
Comment (gwolf2u) [inline laurel_sprout.json:23]: push 14.0

File: laurel_sprout.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+			"common-dt": "",
+			"kernel": "https://github.com/crdroidandroid/android_kernel_xiaomi_laurel_sprout"
```
Comment (gwolf2u) [inline laurel_sprout.json:25]: also here

File: laurel_sprout.txt
Comment (gwolf2u) [inline laurel_sprout.txt:1]: bad filename

**Review by gwolf2u — CHANGES_REQUESTED** [review]

File: laurel_sprout.json (hunk 2 above)
Comment (RainySorcerer) [reply laurel_sprout.json:6]: k sir

**Pull request closed without merging**